app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -5,6 +5,10 @@ def hawkular_client
       hawkular_entrypoint, hawkular_credentials, hawkular_options)
   end
 
+  def metrics
```

Comment:
@yaacov why adding this? (BTW if we really need it you can use something like `alias).